go/cmd/dolt/cli/arg_parser_helpers.go

```diff
@@ -218,6 +219,14 @@ func CreateBackupArgParser() *argparser.ArgParser {
 	return ap
 }
 
+func CreateVerifyConstraintsArgParser() *argparser.ArgParser {
+	ap := argparser.NewArgParser()
+	ap.SupportsFlag(AllFlag, "a", "Verifies constraints against every row.")
```

Comment:
All rows? Or all tables?

Reply:
I'll improve the description, it is all rows. If you specify no tables then all tables are considered...